torch/_functorch/aot_autograd.py

```diff
@@ -2362,6 +2375,8 @@ def create_functionalized_rng_ops_wrapper(func, args, trace_joint=True):
     # At runtime, we pass on the current seed and offset. This is hidden from
     # the user.
     fake_mode = detect_fake_mode()
+    if fake_mode is None:
+        fake_mode = FakeTensorMode(shape_env=None)
```

Comment:
are you sure? Creating a fresh fake tensor mode is typically an antipattern

Reply:
No, it doesn't. Good point! I should make it a nullcontext.